fastlane_core/lib/fastlane_core/analytics/analytic_context.rb

```diff
@@ -0,0 +1,9 @@
+module FastlaneCore
+  class AnalyticContext
```

Comment:
lets rename all `Analytic` to `Analytics`
"Analytics: information resulting from the systematic analysis of data or statistics."